src/server/api-server.js

```diff
@@ -77,7 +78,7 @@ class APIServer extends EventEmitter {
   startListening() {
     return new Promise((resolve, reject) => {
       this._expressServer = this._expressApp.listen(
-        this._port, this._host, () => {
+        this._port, this._serveHost, () => {
```

Comment:
The only reason I'm being picky over this is that I know sysops are generally anti 0.0.0.0 as it opens up the API to the network (which makes sense for a docker container) but not for a tightly controlled network (a.l.a. my work computer).